### DISCUSSION TITLE
rustic v0.9.1

### DISCUSSION THREAD

**Discussion by simonsan** [rustic-rs/rustic#1290] [Announcements]
Hello,

today we are releasing a patch release to rustic, where we added:

- some more filters for you to use to filter your snapshots to your liking
- the `docs` command, that opens the rustic documentation in your default browser
- the documentation as a PDF in the release package

We also improved the performance of the interactive snapshot command.

Enjoy!

# Changelog

### Features

- *(config)* add more filters ([#1263](https://github.com/rustic-rs/rustic/pull/1263))
- *(check)* Allow to only check trees+packs for given snapshots ([#1230](https://github.com/rustic-rs/rustic/pull/1230))
- *(commands)* add a `docs` command to easily access the user, dev and config documentation ([#1276](https://github.com/rustic-rs/rustic/pull/1276))

### Bug Fixes

- *(deps)* update rustic_core and other dependencies and fix merge precedence ([#1282](https://github.com/rustic-rs/rustic/pull/1282))

### Documentation

- *(docs/cli)* improve the descriptions of the CLI commands ([#1277](https://github.com/rustic-rs/rustic/pull/1277))
- *(docs)* update configuration documentation to align with recent changes ([#1280](https://github.com/rustic-rs/rustic/pull/1280))

### Miscellaneous

- *(deps)* upgrade dependencies ([#1289](https://github.com/rustic-rs/rustic/pull/1289))
- add triage label to new issues only if no label has been set when creating it ([#1287](https://github.com/rustic-rs/rustic/pull/1287))
- *(interactive)* use update methods for refreshing snapshots ([#1285](https://github.com/rustic-rs/rustic/pull/1285))

<hr /><em>This discussion was created from the release <a href='https://github.com/rustic-rs/rustic/releases/tag/v0.9.1'>rustic v0.9.1</a>.</em>

**Comment by aawsome** [comment]
A note about the `check` improvements: This are in fact three enhancements:
- You can now (optionally) specify snapshots for which the tree and pack (if `--read-data` is selected) checks are done. This includes using filters or something like `rustic check latest`. 
- The option `--read-data-subset`  is added which allows to only choose a subset of pack files to read (if `--read-data` is selected). Possible choices are `n/m` which allows to completely check all data split up in `m` runs or random selections given by percentage or absolute size.
- When using `--read-data` in combination with a hot/cold repository, the packs which are checked are now warmed-up before.
